10.11/Dockerfile.rhel10

```diff
@@ -11,53 +11,55 @@ FROM ubi10/s2i-core
 #  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
 
 ENV MYSQL_VERSION=10.11 \
+    MYSQL_SHORT_VERSION=1011 \
+    VERSION=${MYSQL_VERSION} \
     APP_DATA=/opt/app-root/src \
     HOME=/var/lib/mysql \
     NAME=mariadb \
-    VERSION=10.11 \
-    SUMMARY="MariaDB 10.11 SQL database server" \
+    VERSION=${MYSQL_VERSION} \
+    SUMMARY="MariaDB ${MYSQL_VERSION} SQL database server" \
     DESCRIPTION="MariaDB is a multi-user, multi-threaded SQL database server. The container \
 image provides a containerized packaging of the MariaDB mysqld daemon and client application. \
 The mysqld server daemon accepts connections from clients and provides access to content from \
 MariaDB databases on behalf of the clients."
 
-LABEL summary="$SUMMARY" \
-      description="$DESCRIPTION" \
-      io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="MariaDB $VERSION" \
+LABEL summary="${SUMMARY}" \
```

Comment:
Is there any reason why you switch from $SUMMARY -> ${SUMMARY} and similar.

The our container fails should meet POSIX criteria, I guess.

This change means that the rest cof the ontainers will have different naming conventions.

Reply:
But it looks like the other containers used this naming. let's have it there.

Reply:
I used the PHP container as the example:
https://github.com/sclorg/s2i-php-container/blob/master/8.2/Dockerfile.rhel9

However even there the style is not fully coherent, so I'd suggest to unify it there.

I find this style much more readable for a human, I don't have a strong preference from the technical POV.

The readibility of this style is IMO highlighted in cases where multiple variables are used next to each other, such as:
```
io.openshift.tags="database,mysql,${NAME},${NAME}${MYSQL_SHORT_VERSION},${NAME}-${MYSQL_SHORT_VERSION}" \
```and similar, as the boundaries of the variable name are now clear.